openstack/cerebro/cerebro-main/values.yaml

```diff
@@ -0,0 +1,20 @@
+image:
+  pullPolicy: IfNotPresent
+  tag: SET-AS-HELM-PARAMETER # This should be set via --set to a git commit hash
+  registry: SET-AS-HELM-PARAMETER
+
+vernemq:
+  vernemq:
+    image:
+      repository: SET-AS-HELM-PARAMETER
+      tag: SET-AS-HELM-PARAMETER  # This should be set via --set to a git commit hash

```

Comment:
As above. Do you maintain this image or is this something public? If the ladder, we've got to fetch it with keppel too.

Reply:
Yes, we maintain this image. All images are stored on Keppel.